src/lib/stitching/gravity/stitching.ts

```diff
@@ -74,6 +81,32 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        formattedEndAt: {
+          fragment: gql`
+            ... on ViewingRoom {
+              startAt
+              endAt
```

Comment:
I think `startAt` and `endAt` are local time (though its ISO8601 value _may_  include UTC as the timezone) and there is a separate `timezone` field to specify the local time timezone.

If Communities viewing rooms always specify UTC as the timezone, this will work. But maybe we want to take the timezone into account when calculating the diff.

Reply:
Added [here](https://github.com/artsy/metaphysics/pull/2382/files#diff-573ae00d2f637334b78f6df4edf48571R108) but not sure if this is the correct way.